### ISSUE TITLE
Listening Interface Ends Up On 0.0.0.0- this normal?

### ISSUE THREAD

**Issue by tteddy623** [louislam/uptime-kuma#2222]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### Description

Reading other posts I see port changing questions but I cannot seem to get the server to exclude other specified devices.  For example, I have 2 NICS eth0 and eth1, one of which has several virtual sub-interfaces i.e. eth0.12.  Kuma's base URL is set for an eth1 address.  Problem is kuma (3001) is found on every interface.  Yes, I can config nftables to just block it but I want to verify that this is normal.  I would expect the base URL to stay on that listening address/subnet rather than 0.0.0.0.

### 👟 Reproduction steps

Specify the primary base URL in Kuma settings 

### 👀 Expected behavior

Kuma runs only on the interface for the broadcast domain specified under the primary base URL

### 😓 Actual Behavior

Kuma is hosted on every interface

### 🐻 Uptime-Kuma Version

1.18.0

### 💻 Operating System and Arch

Pop!_OS 22.04 LTS

### 🌐 Browser

Firefox 105.0.1

### 🟩 NodeJS Version

16.17.1

**Comment by Computroniks** [comment]
The base URL and the address that the server binds to are two different things. The base URL is used client side to ensure that requests for things such as CSS and JavaScript files are correct. The bind address sets exactly which address to listen to. You can use the `UPTIME_KUMA_HOST` or `HOST` environment variables to set the bind address. For more info, see the wiki page https://github.com/louislam/uptime-kuma/wiki/Environment-Variables#server-environment-variables

**Comment by tteddy623** [comment]
After a long sigh and allowing the reality of my stupidity to sink in, I have to apologize for wasting your time.  I'm ultra noob to Git and I've opened the wiki page 100 times but never navigated to the other areas, etc.  I'm at work and found this software extremely useful but I had little time and was going crazy looking for config docs.  I got to the point wondering if the client also functioned as a front end control for server config- is this a nodejs thing? lol   Again, apologies and will take the time to learn the ins-and-outs of the community, and VERY interested in supporting louislam's efforts.  Thank you!
